app/serializers/api/v3/top_profiles_serializer.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V3
+    class TopProfilesSerializer < ActiveModel::Serializer
+      attributes :context_id, :node_id
+      attribute :profile_type
+      attribute :photo_url
+      attribute :year
+      attribute :summary
+
+      def profile_type
+        object.node.node_type.context_node_types.find_by(context_id: object.context_id).profile.name
+      end
+
+      def year
+        object.context.years.max
+      end
+
+      def summary
```

Comment:
yes, this is exactly what we should do! I'm just worried it might be slow, because the top profiles will usually have a lot of data to process. can you test by setting some of the bigger ones as top profiles, look at the top nodes on the sankey in the first and the third column for Brazil-soy, Paraguay-soy and Paraguay-beef, set ~20 top profiles and see how long it takes.

Reply:
You're right, it doesn't look so good; for 9 bigger nodes it takes to load up to ~13 seconds. For comparison, without the summary it's 78.87 ms..

Reply:
maybe we could add `summary` to the top profiles table, that should make it quicker. wdyt @agnessa ?

Reply:
sure! you could also add the year, so that it's clear from which data it was generated